hydromt_wflow/wflow.py

```diff
@@ -417,6 +417,13 @@ def setup_rivers(
         """
         self.logger.info("Preparing river maps.")
 
+        # Check that river_upa threshold is bigger than the maximum uparea in the grid
+        if river_upa > self.grid[self._MAPS["uparea"]].max():
+            raise ValueError(
+                f"river_upa threshold {river_upa} should be larger than the maximum \
```

Comment:
```suggestion
                f"river_upa threshold {river_upa} should be smaller than the maximum \
```

Reply:
I think larger is good here :). Ie I have a basin of 10km2 but my river_upa threshold is 30km2 so I won't create any river cell.
I think I actually made the mistake in the code line 421 to check on smaller and then Piave could not be built.